gui.py

```diff
@@ -66,59 +74,122 @@ def run_sss(self):
             extract(method=self.my_data.method,
```

Comment:
Może puszczenie `extract` w osobnym wątku pozwoliłoby usunąć buga z nieodpalonym gifem?
Dodatkowo - `extract` zwraca AudioWave, więc można zapisać wynik stąd i później odtworzyć. Zaoszczędzi to zapisywania w folderze `temp_files`